### PR TITLE
Update profile dists

### PR DESCRIPTION
**Pull request by apetkau**
This PR implements the following changes:

* Updates `profile_dists` to latest version found in the Docker hub container release tagged `mwells14/gsp:arborator_1.0.0`
* Adds test cases for adjusting `--pd_missing_threshold` (threshold for proportion of samples with missing alleles for a loci beyond which that locus is removed from distance calculations), and `--pd_sample_quality_threshold` (threshold for proportion of loci with missing allele data for a sample before that sample is removed from distance calculations).
* Sets `--pd_outfmt matrix` as the only valid value since the `matrix` format of distances is required for the `gas mcluster` command (`pairwise` is used for `gas call`). I left this as a parameter in case something is updated later, but it is also now hidden from the IRIDA Next UI.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).

### CODE REVIEW

File: tests/pipelines/main.nf.test [hunk 1]
```diff
@@ -150,79 +150,6 @@ nextflow_pipeline {
         }
     }
 
-    test("Full pipeline hashes and missing data") {
```
Comment (apetkau) [inline tests/pipelines/main.nf.test:153]: These tests were moved to the file `tests/pipelines/main_missing_thresholds.nf.test`

**Review by kylacochrane — APPROVED** [review]
This all looks great Aaron 😸

**Review by emarinier — APPROVED** [review]
Looks good. I'd lean towards removing `--pd_outfmt` entirely if there's only one option available, but I don't feel strongly about it. I think it's okay.

**Comment by apetkau** [comment]
Thanks @emarinier. I might consider removing `--pd_outfmt` in another PR.

**Pull request merged**